components/input/input.tsx

```diff
@@ -195,7 +210,7 @@ const Input = React.forwardRef<HTMLInputElement, InputProps>(
                 onClick={clearHandler}
               />
             )}
-            {iconRight && <InputIcon icon={iconRight} {...iconProps} />}
+            {iconRight && <InputIcon icon={iconRight} {...iconRightProps} />}
```

Comment:
miss iconProps？

Reply:
It is in the code, the props of the icon on the left